<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_teacher</class>
 <widget class="QDialog" name="add_teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>教工编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NamelineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TellineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordlineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumberlineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SexlineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>院系</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="YuanxilineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sublineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗圆宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>授课科目</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
